docs/input4MIPs_source-id_CMIP6Plus.html

```diff
@@ -201,28 +218,15 @@
       <td>CMIP6Plus</td>
       <td>CR-CMIP-0-2-0</td>
       <td>None</td>
+      <td>abandoned</td>
+      <td>None</td>
       <td>zebedee.nicholls@climate-resource.com;malte.meinshausen@climate-resource.com</td>
       <td>None</td>
       <td>http://www.tbd.invalid</td>
       <td>CR</td>
       <td>CC BY 4.0</td>
-      <td>abandoned</td>
       <td>0.2.0</td>
     </tr>
-    <tr>
-      <td>11</td>
-      <td>Abandoned: registered but never produced</td>
-      <td>CMIP6Plus</td>
-      <td>SOLARIS-HEPPA-CMIP-4-1</td>
-      <td>None</td>
-      <td>bernd@iaa.es</td>
-      <td>None</td>
-      <td>http://solarisheppa.geomar.de/cmip7</td>
-      <td>SOLARIS-HEPPA</td>
-      <td>CC BY 4.0</td>
-      <td>abandoned</td>
-      <td>4.1</td>
```

Comment:
Is there a reason why we lost this? 4.1 is a dataset that NCAR has been using in prototype simulations - it'd be better to keep it

Reply:
Hmm that shouldn't have happened, good pick up

Reply:
Fixed now I believe.